doctr/utils/geometry.py

```diff
@@ -223,8 +223,10 @@ def rotate_boxes(
     rotated_centers = image_center + np.matmul(centers - image_center, rotation_mat)
     x_center, y_center = rotated_centers[:, 0] / orig_shape[1], rotated_centers[:, 1] / orig_shape[0]
     # Compute rotated boxes
-    rotated_boxes = np.stack((x_center, y_center, _boxes[:, 2], _boxes[:, 3], angle * np.ones_like(_boxes[:, 0]),
-                              _boxes[:, 5]), axis=1)
+    rotated_boxes = np.stack(
+        (x_center, y_center, _boxes[:, 2], _boxes[:, 3],
+         angle * np.ones_like(_boxes[:, 0]) + _boxes[:, 4], _boxes[:, 5]), axis=1
```

Comment:
Good catch, I'd only suggest the following
`angle * np.ones_like(_boxes[:, 0]) + _boxes[:, 4]`
---> `_boxes[:, 4] + angle`